src/index.css

```diff
@@ -1,13 +1,13 @@
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: 'Helvetica Neue', sans-serif, -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
```

Comment:
just wondering, is 'Helvetica' the same as 'Helvetica Neue'?

Reply:
Helvetica is a refined version of Helvetica, includes adjusting character weights, proportions, and spacing to Helvetica in 1957. So is it good to use "Helvetica Neue" instead of "Helvetica" xD